web-app/js/chosen/chosen.jquery.js

```diff
@@ -2,18 +2,19 @@
 Chosen, a Select Box Enhancer for jQuery and Prototype
 by Patrick Filler for Harvest, http://getharvest.com
 
-Version 1.1.0
+Version 1.8.7
```

Comment:
Or should this be done as a part of migration to grails 3?

Reply:
I tested it in a bunch of places and it seems it didn't cause any regression. But I'll revert at the first sign of trouble.